index.html

```diff
@@ -83,136 +84,203 @@ <h2>Accessibility Map for COVID-19 assessment centres</h2>
+                               <span class="fl-mapviz-filter-tooltip"></span>
+                               <label class="fl-mapviz-filter-title" for="toggle-3">Accessible Parking</label>
+                               <label class="fl-checkbox-holder" tabindex="0">
+                                   <input class="fl-mapviz-checkbox visually-hidden" id="toggle-3" type="checkbox"/>
+                                   <span></span>
+                               </label>
+                           </div>
+                           <div class="fl-mapviz-filter fl-mapviz-filter-individual">
+                               <span class="fl-mapviz-filter-icon"></span>
+                               <span class="fl-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                               <span class="fl-mapviz-filter-tooltip"></span>
+                               <label class="fl-mapviz-filter-title" for="toggle-4">Individual Service</label>
+                               <label class="fl-checkbox-holder" tabindex="0">
+                                   <input class="fl-mapviz-checkbox visually-hidden" id="toggle-4" type="checkbox"/>
+                                   <span></span>
+                               </label>
+                           </div>
+                           <div class="fl-mapviz-filter fl-mapviz-filter-wait">
+                               <span class="fl-mapviz-filter-icon"></span>
+                               <span class="fl-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                               <span class="fl-mapviz-filter-tooltip"></span>
+                               <label class="fl-mapviz-filter-title" for="toggle-5">Wait Accommodations</label>
+                               <label class="fl-checkbox-holder" tabindex="0">
+                                   <input class="fl-mapviz-checkbox visually-hidden" id="toggle-6" type="checkbox"/>
+                                   <span></span>
+                               </label>
+                           </div>
                         </div>
-                        <div class="fl-mapviz-filter fl-mapviz-filter-washrooms">
-                            <span class="fl-mapviz-filter-icon"></span>
-                            <span class="fl-mapviz-filter-tooltip-icon fl-mapviz-hoverable"></span>
-                            <span class="fl-mapviz-filter-tooltip"></span>
-                            <label class="fl-mapviz-filter-title" for="toggle-2">Accessible Washrooms</label>
-                            <label class="fl-checkbox-holder" tabindex="0">
-                                <input class="fl-mapviz-checkbox visually-hidden" tabindex="-1" id="toggle-2" type="checkbox" value="true" />
-                                <span></span>
-                            </label>
+                        <div class="fl-mapviz-filter-buttons">
```

Comment:
The change in structure of the markup here has caused the tab focus ring to be strangely misshapen - when keyboard focus is on this button, the bottom part of the ring is now jagged